frontend/src/libs/components/card/styles.module.scss

```diff
@@ -72,6 +71,12 @@
   word-break: break-word;
 }
 
+@media (width >= 900px) {
```

Comment:
we use < sign everywhere, so here we should use it too

Reply:
Would it be ok if I put 899 then? Because without = it will be broken on 900px

Reply:
you can use =, but we need to invert the sign <= will be ok

Reply:
Ohhh, but can I use strict measures then?

Reply:
I can write it like this

`@media (900px <= width) `

And it's working, but it feels so wrong😖. And I used to write it more like this `@media (min-width: 900px)`

Reply:
I also like this option `@media (min-width: 900px)`, but we use this `@media (900px <= width)` in other places in the app

Reply:
Ye, I was asking cause didn't see in the development this way of writing it yet. Ok, I changed it!